pkg/plugin/plugin.go

```
@@ -301,6 +301,15 @@ type Variable struct {
 	Label string `json:"text"`
 }
 
+// trimVariables strips the whitespace that source systems tend to leave around
+// text values, so that dashboard dropdown entries line up with each other.
+func trimVariables(vars []Variable) {
```

golangci-lint (2.12.2)

[error] func trimVariables is unused

(unused)